flags.html

```diff
@@ -4,80 +4,82 @@
         <link rel="stylesheet" href="flags.css">
     </head>
     <body>
-<div id="indonesia" class="flagContainer">
+        <div id="indonesia" class="flagContainer">
 
-</div>
-<div id="columbia" class="flagContainer">
+        </div>
+        <div id="columbia" class="flagContainer">
+            <div class="topSegment yellow"></div>
```

Comment:
a more efficient way to deal with this would be
```
<div id="columbia" class="flagContainer">
    <div class="halfSegment yellow"></div>
     <div class="halfSegment">
         <div class="halfSegment blue"></div>
         <div class="halfSegment red"></div>
     </div>
</div>